packages/dds/tree/src/shared-tree/sharedTreeChangeFamily.ts

```diff
@@ -116,6 +116,7 @@ export class SharedTreeChangeFamily
 								new: innerChange.innerChange.schema.old,
 								old: innerChange.innerChange.schema.new,
 							},
+							isRollback: true,
```

Comment:
I should just have called this `isInverse`

Reply:
Okay, so in this context, `isInverse` describes whether the schema change was generated by an `invert`, and `isRollback` describes whether `invert` is being called by a rollback, whatever that is (I used to know and I have forgotten), and it matters for data changes. 

If you invert a schema change and then invert it back, is `isInverse` true or false for that third schema change?

Reply:
`isRollback` is only true for changes that are only meant to in-order revert a locally applied change so that we can pretend it never happened (this is needed for transaction abort, and for rebasing). These effectively wind back the timeline of changes.

Non-rollback inverses are just new changes that add to the timeline of changes. It just happens that they were generated by inverting some other change but that's largely incidental to how the rest of the system works.

If you invert a schema change and then invert it back, `isInverse` will be and should be true. This is because it was produced by inverting some change.

Reply:
Makes sense to me, thanks :)